golem-cloud-server-base/src/model.rs

```diff
@@ -1172,7 +1176,7 @@ impl TryFrom<golem_api_grpc::proto::golem::template::FunctionResult> for Functio
     ) -> Result<Self, Self::Error> {
         Ok(Self {
             name: value.name,
-            tpe: value.tpe.ok_or("Missing tpe")?.try_into()?,
+            typ: value.tpe.ok_or("Missing tpe")?.try_into()?,
```

Comment:
`Missing typ`?

Reply:
I guess this change is basically fixing OSS version based on the fixes we had in golem cloud from @nicoburniske . Am I right @justcoon ?

Reply:
yes, I applied changes from golem cloud, to be same